lib/rubocop/cop/style/space_after_not.rb

```diff
@@ -16,6 +16,8 @@ class SpaceAfterNot < Cop
         MSG = 'Do not leave space between `!` and its argument.'.freeze
 
         def on_send(node)
+          return if node.keyword_not?
```

Comment:
That's a bit inefficient as we'll end up unpacking the send node twice. Maybe we should just check the name of the send node around the place where we check for !.